app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,23 @@
+class FriendshipsController < ApplicationController
+	
+	def index
+		@user = User.find(params[:id]) || current_user
```

Comment:
I guess we can simply just assign `@user` to `User.find(params[:id])` regardless whether it's the page of the `current_user` we are visiting or not.

Reply:
We also want to have a link to friend list in the navbar,  and there would be no params passed from there.